cermine-impl/src/main/java/pl/edu/icm/cermine/CommandLineOptionsParser.java

```diff
@@ -38,6 +38,9 @@
     public CommandLineOptionsParser() {
```

Comment:
One of the unit tests throws an error because of this class. Also, you should update tests in `pl.edu.icm.cermine.CommandLineOptionsParserTest`, so that the tests check your new options as well.